src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/EndreStudinivå.tsx

```diff
@@ -23,7 +26,12 @@ export const EndreStudienivå: React.FC<{
             value={form.studienivå || ''}
             legend="Studienivå"
             readOnly={!alleFelterKanEndres}
-            onChange={(e) => settStudienivå(e)}
+            onChange={(e) => {
+                settStudienivå(e);
+                if (alder !== undefined && alder >= 22) {
+                    settHarRettTilUtstyrsstipend(SvarJaNei.NEI);
```

Comment:
Jeg er usikker på om vi alltid skal sette denne til NEI hvis saksbehandler ikke har sett eller svart på den. 
Hør gjerne med en funksjonell her

Reply:
Jeg snakket med Nina. Hun sa at den skulle Preutfylles med  Nei. Så jeg trenger ikke å endre noe .

Reply:
Tror det kan ha blitt et missforstand då. 

Min tanke her var at hvis man har høyere utdanning, så burde vi ikke sette "har rett til utstyrsstipend" til nei automatisk. Det som då skjer er at vi lagrer ned at saksbehandler har tatt stilling til noe som den ikke har. 
I det tilfellet burde ikke "har rett til utstyrsstipend" bli vurdert i det hele tatt.

Reply:
Eksemplet med Høyere utdanning er ikke lenger relevant, da valget om utstyrsstipend ikke lenger vises i dette tilfellet.

For Videregående skole skal radioknappen fortsatt prepopuleres med "NEI", dersom brukeren er 21 år eller eldre

Dette har vi implementert her: [2faef78](https://github.com/navikt/tilleggsstonader-sak-frontend/pull/647/commits/2faef78ddd638f938573ca85bd1888a87fe1a799)

Se bilde: 
![image](https://github.com/user-attachments/assets/0b62a3e3-1dab-4d6d-9bb4-bcbb1b2a8bf5)